platform-common-resources/modules.tf

```diff
@@ -1,14 +1,14 @@
 module "create-ingress-nginx" {
   source = "./create-ingress-nginx"
 
-  monitoring_namespace  = var.monitoring_namespace
-  ingress_nginx_version = var.ingress_nginx_version
-  loadbalancer_ip       = var.loadbalancer_ip
-  tls_secret_name       = var.tls_secret_name
-  resource_group        = var.resource_group
+  monitoring_namespace    = var.monitoring_namespace
+  ingress_nginx_version   = var.ingress_nginx_version
+  loadbalancer_ip         = var.loadbalancer_ip
+  tls_secret_name         = var.tls_secret_name
+  publicip_resource_group = var.publicip_resource_group
```

Comment:
When not in 'ARM' context, do we need to provide this var as well ? Or maybe we should create a condition to use old 'resource_group' var when not in arm context ?